### DISCUSSION TITLE
timeout error

### DISCUSSION THREAD

**Discussion by acedinh001** [ultrafunkamsterdam/undetected-chromedriver#1557] [Q&A]
urllib.error.URLError: <urlopen error [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond>
i face this error when running undetected-chromedriver using cellular

**Comment by acedinh001** [comment]
from selenium import webdriver
from selenium.webdriver.chrome.options import Options
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC
from selenium.webdriver.common.keys import Keys
from selenium.webdriver.common.action_chains import ActionChains

import GoogleSearchController.Elements as el

import undetected_chromedriver
from selenium_stealth import stealth
from pprint import pprint

import time

class GoogleSearchController():
    def __init__(self):
        pass
    
    def launch(self):
        chrome_options = webdriver.ChromeOptions()
        chrome_options.add_argument("--disable-notifications")
        chrome_options.add_argument("--disable-infobars")
        chrome_options.add_argument("--mute-audio")
        chrome_options.add_argument("--disable-infobars")
        chrome_options.add_argument("--disable-notifications")
        chrome_options.add_argument("--mute-audio")
        self.driver = webdriver.Chrome(options=chrome_options)
        self.driver.set_page_load_timeout(60)
        self.driver.maximize_window()
        stealth(self.driver,
                languages=["en-US", "en"],
                vendor="Google Inc.",
                platform="Win32",
                webgl_vendor="Intel Inc.",
                renderer="Intel Iris OpenGL Engine",
                fix_hairline=True,
                )
        self.driver.get("https://www.google.com.vn")
        this is mycode

**Reply by jdholtz** [reply]
Where are you using undetected chromedriver? It looks like you import it, but you are actually using the Selenium webdriver and ChromeOptions.
